File: exercises/D203/solution.cpp
#include <iostream> 
using namespace std;
int main() {
    int n;
    cin >> n;
    for (int i = 0; i < 10; i++) {
        for (int j = 1; j <= 10; j++) {
            int k = i * 10 + j;
            if (k / 10 == n || k % 10 == n || k % n == 0) {
                cout << "Clap";
            } else {
                cout << k;
            }
            if (j < 10) {
                cout << " ";
            } else {
                cout << endl;
            }
        }
    }
}

File: exercises/D203/solution.py
n = int(input())
for i in range(10):
    for j in range(1, 11):
        k = i * 10 + j
        if k // 10 == n or k % 10 == n or k % n == 0:
            print('Clap', end='')
        else:
            print(k, end='')
        if j < 10:
            print(' ', end='')
        else:
            print()
